### PR TITLE
kajal/Mar-28

### PR DESCRIPTION
**Pull request by kkaajjaall**

### CODE REVIEW

File: resources/views/components/diploma-form.blade.php [hunk 1]
```diff
@@ -147,3 +156,125 @@
...
+                            numeric: {
+                                min: 8,
+                                message: 'year form must have at least 8 characters',
+                            },
+                        },
+                    },
+                    diploma_year_to: {
+                        validators: {
+                            notEmpty: {
+                                message: '  year to is required',
```
Comment (ithqindia) [inline resources/views/components/diploma-form.blade.php:255]: Empty spaces

File: resources/views/components/diploma-form.blade.php [hunk 2]
```diff
@@ -147,3 +156,125 @@
...
+                            numeric: {
+                                min: 8,
+                                message: ' year to must have at least 8 characters',
+                            },
+                        },
+                    },
+                    completion_status: {
+                        validators: {
+                            notEmpty: {
+                                message: 'Select option is required',
```
Comment (ithqindia) [inline resources/views/components/diploma-form.blade.php:266]: Select option use meaningful message

File: resources/views/components/diploma-form.blade.php [hunk 3]
```diff
@@ -147,3 +156,125 @@
...
+                        },
+                    },
+                    diploma_year_to: {
+                        validators: {
+                            notEmpty: {
+                                message: '  year to is required',
+                            },
+                            numeric: {
+                                min: 8,
+                                message: ' year to must have at least 8 characters',
```
Comment (ithqindia) [inline resources/views/components/diploma-form.blade.php:259]: Empty space

**Review by ithqindia — CHANGES_REQUESTED** [review]

File: resources/views/components/diploma-form.blade.php [hunk 4]
```diff
@@ -115,10 +115,14 @@
...
         <div class="col-md-4 col-sm-12">
-            <label for="inputYearto" class="form-label">Status<span
+            <label for="completion_status" class="form-label">Status<span
                     style="color:red; font-size:20px">*</span></label>
-            <select class="form-select" aria-label="Default select example" name="completion_status">
-                <option selected disabled>- Please select -</option>
+            <select class="custom-select  form-control" 
+             name="completion_status" @error('completion_status') is-invalid @enderror
+            id="completion_status" 
+            value="{{ old('completion_status', $diplomaInformation->completion_status ?? '') }}">
```
Comment (ithqindia) [inline resources/views/components/diploma-form.blade.php:123]: code not formatted

File: resources/views/components/diploma-form.blade.php [hunk 5]
```diff
@@ -131,6 +135,11 @@
                     <option value="Dropped">Dropped</option>
                 @endif
             </select>
+            @error('completion_status')
+            <div class="invalid-feedback">
+                {{ $message }}
+            </div>
+        @enderror
```
Comment (ithqindia) [inline resources/views/components/diploma-form.blade.php:142]: Wrong formatting

File: resources/views/components/diploma-form.blade.php [hunk 6]
```diff
@@ -147,3 +156,128 @@
...
+                                min: 3,
+                                max: 255,
+                                message: 'Diploma langauge must have at least 3 characters and less than 255 characters',
+                            },
+                        },
+                    },
+                    diploma_year_form: {
+                        validators: {
+                            notEmpty: {
+                                message: 'Diploma year form is required',
```
Comment (ithqindia) [inline resources/views/components/diploma-form.blade.php:248]: form -> from

File: resources/views/components/diploma-form.blade.php [hunk 7]
```diff
@@ -147,3 +156,128 @@
...
+                            numeric: {
+                                message: 'Value is not a number',
+                            },
+                        },
+                    },
+                    completion_status: {
+                        completion_status: {
+                        validators: {
+                            notEmpty: {
+                                message: 'Please select the Completion status',
```
Comment (ithqindia) [inline resources/views/components/diploma-form.blade.php:269]: Remove the

**Review by ithqindia — CHANGES_REQUESTED** [review]

File: resources/views/components/ielts-form.blade.php [hunk 8]
```diff
@@ -135,8 +140,136 @@
...
+                            numeric: {
+                                message: 'Value is not a number',
+                            }
+                        }
+                    },
+                    ielts_trf: {
+                        validators: {
+                            notEmpty: {
+                                message: 'Trf is required',
+                            },
```
Comment (ithqindia) [inline resources/views/components/ielts-form.blade.php:231]: TRF

**Review by ithqindia — CHANGES_REQUESTED** [review]

File: resources/views/components/gre-general-form.blade.php [hunk 9]
```diff
@@ -135,3 +135,80 @@ class="form-control @error('gre_general_test_registration_number') is-invalid @e
         </div>
     </div>
 </div>
+
+@section('scripts')
+
```
Comment (ithqindia) [inline resources/views/components/gre-general-form.blade.php:140]: Empty line

**Review by ithqindia — CHANGES_REQUESTED** [review]

File: resources/views/components/gre-general-form.blade.php (hunk 9 above)
Comment (kkaajjaall) [reply resources/views/components/gre-general-form.blade.php:140]: at starting i made some changes in this form but now gre-general-form is done by vivek .

Comment (ithqindia) [reply resources/views/components/gre-general-form.blade.php:140]: That does not matter who has done what, you need to change it.

**Comment by ithqindia** [comment]
Merged locally

**Pull request closed without merging**